File: bench/bench_calls.lua
local function id(x) return x end
local t0 = os.clock()
local s = 0
for i = 1, 1000000 do s = s + id(i) end
local t1 = os.clock()
print(string.format("lua    calls 1M = %d  time: %.4fs", s, t1-t0))

File: bench/bench_calls.py
import time
def id_(x): return x
t0 = time.process_time()
s = 0
for i in range(1, 1_000_001): s += id_(i)
t1 = time.process_time()
print(f"python calls 1M = {s}  time: {t1-t0:.4f}s")
